tools/scripts/localstack/start.py

```
@@ -78,6 +205,26 @@ def main() -> int:
         logger.error("Failed to start LocalStack container:\n%s", result.stderr)
         return result.returncode
     logger.info("LocalStack container started successfully with ID: %s", result.stdout.strip())
+
+    endpoint = _wait_for_localstack_ready(args.name)
+    if not endpoint:
+        _dump_container_diagnostics(args.name)
+        # Clean up the container we started so a retry is not blocked by a stale name.
+        subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)